scripts/uart_terminal/README.rst

```diff
@@ -0,0 +1,66 @@
+.. _uart_terminal_script:
+
+UART terminal script
+####################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The UART terminal script can be used with any setup that uses several development kits to start a terminal connection to all DKs at the same time.
+This can save time needed to manually establish multiple connections, which speeds up :ref:`gs_testing`.
+
+For example, when working with the :ref:`nrf53_audio_app` in the BIS mode with several receivers, you can use the script instead of manually opening a terminal connection to each DK.
+
+Overview
+********
+
+Once started, the script checks for existing serial connections by running the ``nrfjprog --com`` command in the background.
+It then starts a terminal connection for each DK's COM port with the number corresponding to the one specified in the :file:`get_serial_ports.py` file.
+By default, the terminal connection is opened for the virtual COM port (VCOM) of the application core (``VCOM0``).
```

Comment:
This can be used for several kits, so perhaps just mention "VCOM0"

Reply:
Yes true I will remove the "application core"

Reply:
Done